src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -102,7 +100,9 @@ const SetAclPermissionForm = () => {
       await setDocAclPermission(session, docType, permissions, podUsername);
```

Comment:
Would likely be fixed to use podUrl in the future for the whole form, similar to what it was done for setDocContainerAclPermission.